pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -2,31 +2,34 @@
 #include "bpf_telemetry.h"
 #include "bpf_builtins.h"
 #include "bpf_tracing.h"
-#include "tracer.h"
-
-#include "protocols/protocol-classification.h"
-#include "tracer-events.h"
-#include "tracer-maps.h"
-#include "tracer-stats.h"
-#include "tracer-telemetry.h"
-#include "sockfd.h"
-#include "tcp-recv.h"
-
 #include "bpf_endian.h"
-#include "ip.h"
-#include "ipv6.h"
-#include "port.h"
 
+#include <linux/err.h>
+#include <linux/socket.h>
 #include <net/inet_sock.h>
 #include <net/net_namespace.h>
+#include <net/sock.h>
 #include <net/tcp_states.h>
+#include <uapi/linux/if_ether.h>
 #include <uapi/linux/ip.h>
 #include <uapi/linux/ipv6.h>
 #include <uapi/linux/ptrace.h>
 #include <uapi/linux/tcp.h>
 #include <uapi/linux/udp.h>
-#include <linux/err.h>
 
+#include "conn-tuple.h"
+
+#include "tracer.h"
+#include "protocols/protocol-classification.h"
+#include "tracer-events.h"
```

Comment:
Any particular reason the include order had to change here? It's hard to see exactly what was added and removed since the order changed

Reply:
Sorry about that. There should be no new include files for prebuilt. However, the big change is that they are moved after the system includes above. This is because we can't use the system headers in the common header files since they are not available for CO-RE compiles. So we have to include them first so that the common files like `tracer.h` build on prebuilt and runtime compiled.